### ISSUE TITLE
[Backport] `cobbler buildiso` artifacts are not EFI-bootable

### ISSUE THREAD

**Issue by SchoolGuy** [cobbler/cobbler#3744]
### Original feature issue

- Issue: #3381
- PR: #3357

### Target release

- [x] release33
- [ ] release32
- [ ] release30

### Reason

Alignment with Uyuni/SUMA.

The backport is already done but just needs to be submitted upstream: https://github.com/openSUSE/cobbler/pull/34

I have an untested commit rebased onto 3.3.5 already locally available. I will attempt to submit the backport as soon as possible.

**Comment by SchoolGuy** [comment]
This needs to be backported after #3747 since the split of the tests happened in there.
